src/WorkOS.net/Services/_common/_interfaces/ListOptions.cs

```diff
@@ -24,5 +24,11 @@ public class ListOptions : BaseOptions
         /// </summary>
         [JsonProperty("limit")]
         public long? Limit { get; set; }
+
+        /// <summary>
+        /// Order of records returned by ascending or descending order.
+        /// </summary>
+        [JsonProperty("order")]
+        public string Order { get; set; }
```

Comment:
Could we use a more strongly typed representation by making this an enum?

```cs
[JsonConverter(typeof(StringEnumConverter))]
public enum OrderBy
{
    [EnumMember(Value = "asc")]
    Asc,
    [EnumMember(Value = "desc")]
    Desc
}
```

I can't remember if enums are value types, so we would either have this:

```suggestion
        public PaginationOrder? Order { get; set; }
```

Of if they are reference types then we won't be able to mark it as optional (without nullable reference types, which we don't currently have enabled):

```suggestion
        public PaginationOrder Order { get; set; }
```

Reply:
Looks like they are value types:
 https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/enum
 
 Added as enum!